phpBB/composer.json

```diff
@@ -33,7 +33,7 @@
 		"marc1706/fast-image-size": "1.1.*",
 		"paragonie/random_compat": "^1.2",
 		"patchwork/utf8": "1.1.*",
-		"s9e/text-formatter": "^0.4.2",
+		"s9e/text-formatter": "^0.5.0",
```

Comment:
Do you want to restrict to 0.5.* or 0.* ?
If it is 0.5.*, then use `~0.5`. Otherwise `^0.5`